development/webpack/utils/cli.ts

```diff
@@ -394,13 +409,19 @@ function getOptions(
 /**
  * Returns a string representation of the given arguments and features.
  *
- * @param args
- * @param features
+ * @param args - The parsed CLI arguments
+ * @param features - The active and available features
+ * @param resolvedEnvironment - The resolved MetaMask environment
  */
-export function getDryRunMessage(args: Args, features: Features) {
+export function getDryRunMessage(
+  args: Args,
+  features: Features,
+  resolvedEnvironment?: string,
+) {
   return `🦊 Build Config 🦊
 
-Environment: ${args.env}
+Environment (--env): ${args.env}
+Target Environment: ${resolvedEnvironment ?? args.targetEnvironment ?? '(auto-detected)'}
```

Comment:
Given that `resolvedEnvironment` is always set, the later two cases are dead code.

Maybe we can do something like this to ensure `(auto-detected)` is displayed if we auto-detected the resolved environment though:

```suggestion
Target Environment: ${resolvedEnvironment}${args.targetEnvironment ? '' : ' (auto-detected)'}
```